apps/deploy-app.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: "Deploy"
+sidebarTitle: "Deploy your App"
+description: "Deploy your Apps to production with Hypermode"
+"og:title": "Deploy - Apps"
+---
+
+Once you've developed and tested your App locally, you're ready to deploy it to
+production on Hypermode. This guide walks you through setting up automatic
+deployment for your Modus app.
+
+## Prerequisites
+
+Before deploying, ensure you have:
+
+- A completed Modus app (see the [Develop guide](/apps/develop-app) for setup)
+- A GitHub account and repository
+- Your app configured in the Hypermode console (see the
+  [Create Your App guide](/apps/create-app))
+
+## Automatic deployment via GitHub Actions
+
+Add a GitHub Actions workflow to your repository for automatic deployments.
+
+Create `.github/workflows/ci-modus-build.yml`:
```

Comment:
We should reference ci-modus-build.yml from modus repo. So we can maintain the file easily at one place.
For example I'm using go 1.24 and tinygo 0.37 but this doc mentions 1.23 and 0.34.

Reply:
@rderbier can you link to where you are referring to?

Reply:
Updated to work with latest 0.38, and link to modus recipes.